mautic/mautic
# Removing unused browser notifications

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [cleanup]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-430



#### Description:

https://github.com/mautic/mautic/pull/352 introduced browser notifications in 2015 but they were never used. The PR even states:

> It is not actually used anywhere yet.

I guess it was experimental and it was never picked up again. Now when we are updating all JS libraries for Mautic 5 this is causing troubles as it is outdated and the library was abandoned.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check that browsing the Mautic administration causes no troubles and there are no JS errors.
